Cargo.toml

```diff
@@ -32,26 +32,35 @@ maintenance = { status = "actively-developed" }
 # Disable CORS Check: This is UNSAFE in general and should only be used in debug builds.
 unsafe-disable-cors = []
 
+# network module
+network = ["manta-util"]
+
+# default dependencies and all modules
+default = ["argon2", "async-std", "chrono", "derivative", "dirs-next", "futures", "http-types",
+  "manta-accounting", "manta-crypto", "manta-parameters", "manta-pay", "manta-util", "parking_lot",
+  "password-hash", "reqwest", "secrecy", "serde_json", "subtle", "tide", "tokio", "network"]
```

Comment:
Why do this kind of feature selection? I don't think you need either of these two features here.